pkg/daemon/discover/discover.go

```diff
@@ -51,18 +52,30 @@ var (
 	cmName          string
 	cm              *v1.ConfigMap
 	udevEventPeriod = time.Duration(5) * time.Second
+	useCVInventory  bool
 )
 
-func Run(context *clusterd.Context, probeInterval time.Duration) error {
+type CephVolumeInventory struct {
+	Path            string          `json:"path"`
+	Available       bool            `json:"available"`
+	RejectedReasons json.RawMessage `json:"rejected_reasons"`
+	SysAPI          json.RawMessage `json:"sys_api"`
+	LVS             json.RawMessage `json:"lvs"`
+}
+
+func Run(context *clusterd.Context, probeInterval time.Duration, useCV bool) error {
 	if context == nil {
 		return fmt.Errorf("nil context")
 	}
-	logger.Infof("device discovery interval is %s", probeInterval.String())
+	logger.Debugf("device discovery interval is %s", probeInterval.String())
+	logger.Debugf("use ceph-volume inventory is %t", useCV)
 	nodeName = os.Getenv(k8sutil.NodeNameEnvVar)
 	namespace = os.Getenv(k8sutil.PodNamespaceEnvVar)
 	cmName = k8sutil.TruncateNodeName(LocalDiskCMName, nodeName)
+	useCVInventory = useCV
```

Comment:
Why do we need this new variable? Isn't `useCV` enough?

Reply:
`useCVInventory` is a module var , `usecv` is just the local var used to pass the cli parameter.
`useCVInventory` is needed in `probeDevices` function, I though that it will be more smart to use a global variable instead of pass this value from function to function.